Makefile

```diff
@@ -23,5 +23,7 @@ clean:
 	@rm -rf node_modules
 	@echo "Done.\n"
 
+test:
```

Comment:
I think `test` should have a `packages` dependency. If I make a fresh install and in the first time I do `make test`, it will crash looking for `mocha` and other packages.